scripts/ninja.js

```diff
@@ -1057,7 +1057,10 @@ ${ninjaQuickBuidList([
       x !== "belt.ml" &&
       x !== "belt_internals.mli" &&
       x !== "node.ml" &&
-      (x.endsWith(".ml") || x.endsWith(".mli")) &&
+      (x.endsWith(".ml") ||
+        x.endsWith(".mli") ||
+        x.endsWith(".res") ||
+        x.endsWith(".resi")) &&
```

Comment:
I don't have much background knowledge about the build system of ReScript. Please review this change.

Reply:
This is fine.